### ISSUE TITLE
Alignment of fov axes

### ISSUE THREAD

**Issue by maxnoe** [gammapy/gammapy#3674]
**Is your feature request related to a problem? Please describe.**

Field of view coordinates can be aligned with local or sky fixed coordinates. 

Current `Background3D` can either be aligned along local or skyfixed coordinates, in GADF, this is stored in the `FOVALIGN` header.

The docs state: 

> To resolve this ambiguity, we propose a header key FOVALIGN={ALTAZ,RADEC}, specifying which definition of field-of-view coordinates is used. If the key is not present, FOVALIGN=ALTAZ should be assumed as default.

However, in gammapy, the default is `RADEC`: https://github.com/gammapy/gammapy/blob/ba7d377bf48bd53d268d2dc14be1c1eb013a1e42/gammapy/makers/map.py#L152

Side note: `FOVALIGN` is a required keyword for `BKG3D`, so the discussion about a default value is kind of mute...

**Describe the solution you'd like**

I think this is a central property of IRFs that are not radially symmetric and there should probably just be a property on those IRFs like `fov_alignment`, comparably to e.g. `is_pointlike`. 

At the moment, this only affects `Background3D`, but there is a long standing issue in GADF about adding non-radially symmetric versions also of the other IRFs.

**Comment by adonath** [comment]
I have never thought about this, but I think ideally this could even be handled as a custom coordinate frame? Such that we get a seamless integration with `astropy.coordinates` as done here https://github.com/cta-observatory/ctapipe/blob/master/ctapipe/coordinates/nominal_frame.py#L31

**Comment by maxnoe** [comment]
@adonath yes, exactly, this is the `FOV_ALIGN=ALTAZ` frame, a second frame could be made by changing the required frame of the origin coordinate.

**Comment by bkhelifi** [comment]
Hi all,
Concerning the `Background3D`, the closed PR #4373 seems to have solved the issue. As we do not yet have such coordinate frame for the MC IRFs, should we close this issue for now?
